collector/lib/SelfChecks.cpp

```diff
@@ -0,0 +1,50 @@
+#include "SelfChecks.h"
+
+#include <cstdint>
+#include <unistd.h>
+
+#include <sys/wait.h>
+
+#include "Logging.h"
+#include "Utility.h"
+
+namespace collector {
+
+namespace self_checks {
+
+const char* kSelfChecksExePath = "/usr/local/bin/self-checks";
+const char* kSelfChecksName = "self-checks";
+const uint16_t kSelfCheckServerPort = 1337;
```

Comment:
What about some more exotic port number, to reduce chances of collision with some other apps?